### PR TITLE
Version 1.2

### PR DESCRIPTION
**Pull request by QuintinWillison**
Draft for now, visibility of what's coming for 1.2.

### CODE REVIEW

File: Ably.podspec [hunk 1]
```diff
@@ -24,4 +24,5 @@ Pod::Spec.new do |s|
   s.dependency 'msgpack', '0.3.1'
   s.dependency 'ULID', '1.1.0'
   s.dependency 'SAMKeychain', '1.5.3'
+  s.dependency 'DeltaCodec', '1.1.0'
```
Comment (QuintinWillison) [inline Ably.podspec]: Hi @ricardopereira ... I've [raised an issue](https://github.com/ably/delta-codec-cocoa/issues/8) against `delta-codec-cocoa` for you to rename this. Once you've done so then could you please use that new pod in this codebase. Happy for you to commit straight against this integration branch. Thanks.

File: Cartfile [hunk 2]
```diff
@@ -1,4 +1,5 @@
 github "ably-forks/SocketRocket" == 0.5.2-ably-7
+github "ably/delta-codec-cocoa" == 1.0.0
```
Comment (QuintinWillison) [inline Cartfile]: @ricardopereira should this be `1.1.0` to match latest tag and the pod?

**Comment by QuintinWillison** [comment]
Hi @ricardopereira ... I was expecting you would rebase this branch on master and then force push it up, therefore I wasn't expecting a merge (see https://github.com/ably/ably-cocoa/pull/982/commits/9a59b098f11a2e96743e9b64b7da92a2ad8cd23e).

Was that intentional?

**Comment by ricardopereira** [comment]
@QuintinWillison I did merge, sorry. I'll rebase instead.

**Pull request merged**